src/applications/disability-benefits/all-claims/pages/newDisabilityFollowUp.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { createSelector } from 'reselect';
+import dateUI from 'us-forms-system/lib/js/definitions/date';
+
+import { getNewDisabilityName } from '../utils';
+import disabilityLabels from '../content/disabilityLabels';
+
+import fullSchema from '../config/schema';
+
+const {
+  cause,
+  primaryDisability,
+  mistreatmentDescription,
+  disabilityStartDate
+} = fullSchema.properties.newDisabilities.items.properties;
+
+export const disabilityNameTitle = ({ formData }) => {
+  return (
+    <legend className="schemaform-block-title schemaform-title-underline">{getNewDisabilityName(formData.diagnosticCode)}</legend>
+  );
+};
+
+export const uiSchema = {
+  newDisabilities: {
+    items: {
+      'ui:title': disabilityNameTitle,
+      cause: {
+        'ui:title': '',
+        'ui:widget': 'radio',
+        'ui:options': {
+          labels: {
+            NEW: 'My disability was caused by—or got worse because of—an injury or exposure during my service in the military.',
```

Comment:
This is probably just my misunderstanding, but `NEW` is also for existing? I.e. not new?

Reply:
I pulled this from some other status list that looked close to what we needed for this question. All of the disabilities are new in here in that the user is entering them for the first time. These questions don't show up for rated disabilities.